charts/databend-query/templates/configmap.yaml

```diff
@@ -53,6 +53,32 @@ data:
       [log.stderr]
         on = {{ .Values.config.log.stderr.enabled }}
         level = {{ .Values.config.log.stderr.level | default "WARN" | quote }}
+      [log.query]
+        on = {{ .Values.config.log.query.enabled }}
+        otlp_endpoint = {{ .Values.config.log.query.otlp_endpoint | quote }}
+        otlp_protocol = {{ .Values.config.log.query.otlp_protocol | default "http" | quote }}
+      {{- with .Values.config.log.query.otlp_labels }}
+      [log.query.otlp_labels]
+        {{- if .tenant }}
```

Comment:
We should not specify `tenant` and `warehouse` here, just assuming labels is a general map.